spec/onchain.tex

```diff
@@ -315,9 +315,67 @@ \subsection{CollectCom Transaction}\label{sec:collect-tx}
 		$\stOpen$ head output.}\label{fig:collectComTx}
 \end{figure}
 
+\subsection{\red{Increment Transaction}}\label{sec:increment-tx}
+
+\noindent The \mtxIncrement{} transaction (Figure~\ref{fig:incrementTx}) allows a party to
+add a UTxO to an already open head and consists of:
+
+\begin{itemize}
+	\item one input spending from $\nuHead$ holding the $\st$ with $\datumHead$,
+	\item one output paying to $\nuHead$ with value $\valHead'$ and
+		  datum $\datumHead'$,
+    \item one or more inputs spent in this transaction $i_{1} \dots i_m$ where $m$ is the number of inputs to include and $\forall i \in \{i_{1} \dots i_{m}\}, i \notin I_{head}$.
+\end{itemize}
+
+\noindent The state-machine validator $\nuHead$ is spent with
+$\redeemerHead = (\mathsf{increment}, \xi)$, where $\xi$ is a multi-signature of
```

Comment:
Must: include a list of output references (inputs) in the redeemer to resolve the UTxO we want to consider (the question mark we put on the transaction traces + red annotation how to work around that)